README.md

```diff
@@ -13,4 +13,4 @@ This repository contains automated deployment template for Laravel Envoy. The de
 
 ## License
 
-The MIT License (MIT). Please see [License File](/LICENSE) for more information.
+This code is licensed under the  [MIT license](http://choosealicense.com/licenses/mit/). Please see [License File](/LICENSE) for more information.
```

Comment:
```
Please see the [license file](LICENSE) for more information..
```

* Add "the"
* Remove title casing on "license file"
* Remove unnecessary / before LICENSE